doc/ref_arch/kubernetes/chapters/chapter06.md

```diff
@@ -10,9 +10,46 @@
 
 <a name="6.1"></a>
 ## 6.1 Introduction
+The adoption of cloud-native allows development teams to move fast, deploy software efficiently, and operate at an unprecedented scale. 
```

Comment:
Can we delete "unprecedented" from "unprecedented scale."

Reply:
Fixed